lib/manager/components/FeedSourceTableRow.js

```diff
@@ -392,8 +392,8 @@ class FeedActionsDropdown extends PureComponent<Props> {
   }
 
   /**
-   * This method is extracted variable generation which had to be split
-   * out of the render function to appease the eslint complexity rule
+   * This method is variable generation which used to live in the render method,
```

Comment:
Sorry I should have mentioned its the term "variable generation" that didn't make sense to me.